Library/Homebrew/cmd/update-bash.sh

```diff
@@ -303,6 +303,9 @@ EOS
       odie "Git must be installed and in your PATH!"
     fi
   fi
+  export GIT_TERMINAL_PROMPT="0"
+  export GIT_ASKPASS="false"
+  export GIT_SSH="$HOMEBREW_LIBRARY/ENV/scm/ssh"
```

Comment:
>  $GIT_SSH_COMMAND takes precedence over $GIT_SSH, and is interpreted
           by the shell, which allows additional arguments to be included.
           $GIT_SSH on the other hand must be just the path to a program
           (which can be a wrapper shell script, if additional arguments are
           needed).